### DISCUSSION TITLE
[zlib-ng] how to use GZFILEOP?

### DISCUSSION THREAD

**Discussion by recsater** [microsoft/vcpkg#34886] [Q&A] [answered]
@luncliff 

I linked and used the zlib-ng package in my project as per your previous [answer](https://github.com/microsoft/vcpkg/discussions/19991#discussioncomment-1403471), but many of the features enabled by definition don't seem to work.

I have attached a gif of the situation. Any help would be greatly appreciated.

![Peek 2023-11-03 16-13](https://github.com/microsoft/vcpkg/assets/16009827/ab84a3c6-93f6-4380-a0e7-f85c59f987ba)

**Comment by autoantwort** [comment]
The function definitions are guarded by `#ifdef WITH_GZFILEOP`

**Reply by autoantwort** [reply]
I think this is a upstream bug. Will create a PR

**Reply by autoantwort** [reply, marked as answer]
You can try #34903
